docs/pages/video.js

```diff
@@ -477,3 +214,11 @@ function Example() {
     </Page>
   );
 }
+
+export async function getStaticProps(): Promise<{| props: {| generatedDocGen: DocGen |} |}> {
+  const generatedDocGen = await docgen({ componentName: 'Video' });
+
+  return {
+    props: { generatedDocGen: deepCloneReplacingUndefined(generatedDocGen) },
```

Comment:
Why are we using this one here? `deepCloneReplacingUndefined`

Reply:
It's a problem with (most) class components: the way some methods are parsed yields `undefined` values, which aren't valid JSON. So then when that tree is serialized, it throws. I'm not sure if we'll have this problem with any other components, but it's necessary for Masonry and Video (Image seems fine).